x-pack/test/functional/services/observability/alerts/rules_page.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export function ObservabilityAlertsRulesProvider({ getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+
+  const getManageRulesPageHref = async () => {
+    const manageRulesPage = await testSubjects.find('manageRulesPage');
+    return manageRulesPage.getAttribute('href');
+  };
+
+  const clickCreateRuleButton = async () => {
+    const createRuleButton = await testSubjects.find('createRuleButton');
+    return createRuleButton.click();
```

Comment:
I haven't run these tests but want to point out that in many cases when clicking to create something we need to wait for some indication that it completed.  For example, there might be a loading indicator.  While a test might pass without it, it can cause flakiness.  Tests can also fail on Cloud if there is a bit more latency and the action doesn't complete very quickly. 

You can simulate this to some extent by running the tests locally with `TEST_THROTTLE_NETWORK=1` as described here https://www.elastic.co/guide/en/kibana/master/development-tests.html#_running_functional_tests